box2d.pro

```diff
@@ -11,11 +11,11 @@ contains(QT_CONFIG, reduce_exports): CONFIG += hide_symbols
 
 # Uncomment the line below to compile qml-box2d plugin with Box2D library, installed in OS
 # or pass the variable to qmake in command line:
-# qmake DEFINES+=BOX2D_SYSTEM
```

Comment:
The idea to use DEFINES was that compiler passes it to C preprocessor and you can use it in C++ code. But yes, I think that not so important so CONFIG looks good here.